src/pages/Version.tsx

```diff
@@ -0,0 +1,157 @@
+import { useState, useEffect } from "react";
+import { useLazyQuery } from "@apollo/client";
+import { useParams, Redirect } from "react-router-dom";
+import { BreadCrumb } from "components/Breadcrumb";
+import { PatchAndTaskFullPageLoad } from "components/Loading/PatchAndTaskFullPageLoad";
+import { PageTitle } from "components/PageTitle";
+import { PatchStatusBadge } from "components/PatchStatusBadge";
+import {
+  PageWrapper,
+  PageContent,
+  PageLayout,
+  PageSider,
+} from "components/styles";
+import { pollInterval } from "constants/index";
+import { commitQueueAlias } from "constants/patch";
+import { getCommitQueueRoute, getPatchRoute } from "constants/routes";
+import { useToastContext } from "context/toast";
+import {
+  VersionQuery,
+  VersionQueryVariables,
+  IsPatchConfigurableQuery,
+  IsPatchConfigurableQueryVariables,
+} from "gql/generated/types";
+import { GET_VERSION, GET_IS_PATCH_CONFIGURED } from "gql/queries";
+import { usePageTitle, useNetworkStatus } from "hooks";
+import { PageDoesNotExist } from "pages/404";
+import { validatePatchId } from "utils/validators";
+import { BuildVariants } from "./version/BuildVariants";
+import { ActionButtons } from "./version/index";
```

Comment:
Does it still make sense to show the "Reconfigure tasks/variants" link for mainline commit versions or should it only be meant for Patches?

Reply:
+1, should only be for patches

Reply:
Nice catch ive disabled the option to select it for non mainline commits